libcontainer/configs/validate/validator.go

```diff
@@ -7,7 +7,7 @@ import (
 	"strings"
 
 	"github.com/opencontainers/runc/libcontainer/configs"
-	"github.com/opencontainers/runc/libcontainer/selinux"
+	selinux "github.com/opencontainers/selinux/go-selinux"
```

Comment:
yup, there shouldn't be the need of aliasing if `go-selinux` exports `selinux` package afaict.